lib/dfid-transition/patch/rummager/base.rb

```diff
@@ -0,0 +1,26 @@
+require 'dfid-transition/patch/patcher'
+
+module DfidTransition
+  module Patch
+    module Rummager
+      class Base < Patcher
+        def mutate_schema
+          raise NotImplementedError
+        end
```

Comment:
No need for `mutate_schema` here? It's in the base class `Patcher#mutate_schema`.